docs/sdk-and-tools/indices/scresults.md

```diff
@@ -0,0 +1,56 @@
+| code              | The code holds the code of the smart contract result.                                                                                                          |
+| data              | The data field holds additional information for a smart contract result. It can contain a simple message, a function call, an ESDT transfer payload, or so on. |
+| prevTxHash        | The prevTxHash holds the hex encoded hash of the previous transaction.                                                                                         |
+| originalTxHash    | The originalTxHash holds the hex encoded hash of the transaction that generated the smart contract result.                                                     |
+| callType          | The callType field holds the type of smart contract call that is done through the smart contract result.                                                       |
+| codeMetaData      | The codeMetaData field holds the code metadata.                                                                                                                |
+| returnMessage     | The returnMessage field holds the message that is returned by a smart contract in case of error.                                                               |
+| timestamp         | The timestamp field represents the timestamp of the block in which the smart contract result was executed.                                                     |
+| status            | The status field holds the execution state of the smart contract result. The execution state can be `pending` or `success`.                                    |
+| tokens            | The tokens field contains a list of ESDT tokens that are transferred based on the data field.                                                                  |
+| esdtValues        | The esdtValues field contains a list of ESDT values that are transferred based on the data field.                                                              |
+| receivers         | The receivers field contains a list of receiver addresses in case of ESDTNFTTransfer or MultiESDTTransfer.                                                     |
+| receiversShardIDs | The receiversShardIDs field contains a list of receiver addresses shard IDs.                                                                                   |
+| operation         | The operation field represents the operation of the smart contract result based on the data field.                                                             |
+| function          | The function field holds the name of the function that is called in case of a smart contract call.                                                             |
+| originalSender    | The originalSender field holds the address of the original sender.                                                                                             |
```

Comment:
The originalSender field holds the sender's address of the original transaction.

Reply:
done